crates/foundry/forge/src/runner.rs

```diff
@@ -795,13 +745,18 @@ impl<'a> ContractRunner<'a> {
                         Ok(call_sequence) => {
                             if !call_sequence.is_empty() {
                                 // Persist error in invariant failure dir.
-                                if let Err(err) = foundry_common::fs::create_dir_all(failure_dir) {
-                                    error!(%err, "Failed to create invariant failure dir");
-                                } else if let Err(err) = foundry_common::fs::write_json_file(
-                                    failure_file.as_path(),
-                                    &call_sequence,
-                                ) {
-                                    error!(%err, "Failed to record call sequence");
+                                if let Some(failure_dir) = failure_dir {
+                                    let failure_file = failure_file.expect("failure file ");
```

Comment:
```suggestion
                                    let failure_file = failure_file.expect("failure file");
```

Typo?

Reply:
Thanks, missed some text there after the space. Fixed in [ab94d0b](https://github.com/NomicFoundation/edr/pull/658/commits/ab94d0b77756dbc8b1caa6fdadb7db918b2fe2f6)